### PR TITLE
Show posts

### PR DESCRIPTION
**Pull request by tuananhhoangit**
------------------------------------------
[x] Build Fail!
------------------------------------------

[x] reek: failed
[o] rspec: success
[o] brakeman: success
[o] bundle-audit: success
[x] rails_best_practices: failed
[o] scss-lint: success
[o] eslint: success
[o] rubocop: success

### CODE REVIEW

**Comment by tuananhhoangit** [comment]
ready

File: app/assets/stylesheets/color.scss [hunk 1]
```diff
@@ -8,3 +8,8 @@ $line-grey: #e5e5e5;
 $devine-grey: #aeaeae;
 $btn-background: #f6f7f9;
 $btn-hover: #dddfe2;
+$success-green: #3c0;
+$success-green-bg: #ecffe6;
+$danger-red: #b30000;
+$danger-red-bg: #ffe6e6;
+
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/color.scss]: remove empty line

File: app/models/post.rb [hunk 2]
```diff
@@ -6,4 +6,20 @@ class Post < ApplicationRecord
   belongs_to :user
 
   scope :posts_sort, ->{order created_at: :desc}
+
+  mount_uploader :picture, PictureUploader
+
+  validates :user_id, presence: true
```
Comment (huongnguyenmta) [inline app/models/post.rb]: `validates :user, presence: true`

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -13,6 +13,14 @@
 
   <body>
     <%= render "layouts/header" %>
+    <% flash.each do |message_type, message| %>
```
Comment (huongnguyenmta) [inline app/views/layouts/application.html.erb]: C move đoạn này vào `views/shared/` rồi render ở đây nhé

File: app/views/posts/_post.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<div class="tech-no">
+  <!-- technology-top -->
```
Comment (huongnguyenmta) [inline app/views/posts/_post.html.erb]: remove this comment

**Comment by huongnguyenmta** [comment]
commented

**Comment by tuananhhoangit** [comment]
ready

**Pull request merged**